build.ps1

```diff
@@ -23,6 +26,10 @@ function initialize_vs {
     try {
         Write-Log "Initializing Visual Studio environment"
         $vs_instances = Get-VSSetupInstance
+        if (-not $vs_instances -or $vs_instances.Length -eq 0) {
+            Write-Log "Error: No Visual Studio instances found"
```

Comment:
"Visual Studio Build Tools" would be better than just "Visual Studio", since Visual Studio" implies that you need to install the IDE (which is not the case).